src/libraries/System.Data.OleDb/src/OleDbException.NoCOMWrappers.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace System.Data.OleDb
+{
+    public sealed partial class OleDbException
+    {
+        internal static OleDbException CreateException(UnsafeNativeMethods.IErrorInfo errorInfo, OleDbHResult errorCode, Exception? inner)
```

Comment:
Can't we merge the two `IErrorInfo` definitions?

Reply:
I was parroting decision with `IPicture` where separate limited interface was created. I though that goal is also to avoid COM attributes. `IErrorInfo` fairly small, and only one method is missing so maybe I can merge if needed. Please advise.

Reply:
I believe COM Attributes are being pruned when possible - https://github.com/dotnet/runtime/pull/54056. Since these interfaces are also both internal I think we can make the definition work for both without too much hassle.